docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -11,15 +11,14 @@ menu:
 type: docs
 ---
 
-Yugabyte’s distributed architecture enables database clusters (also referred to as universes) to have extremely high availability.
+YugabyteDB's distributed architecture enables database clusters (also referred to as universes) to have extremely high availability.
```

Comment:
This is unrelated to YBA HA and not sure we should even mention it here. WDYT? 
I know you are just editing existing paragraph.

Reply:
I agree. Made an edit to the next paragraph as well.